src/containers/course-section/CourseSectionContainer.tsx

```diff
@@ -45,26 +45,19 @@ import {
   CourseResources
 } from '~/types'
 import { styles } from '~/containers/course-section/CourseSectionContainer.styles'
+import CourseSectionsList from '../course-sections-list/CourseSectionsList'
 
-interface SectionProps {
+interface SectionProps
+  extends Pick<
+    Parameters<typeof CourseSectionsList>[0],
```

Comment:
can you explain how ```
extends Pick<
    Parameters<typeof CourseSectionsList>[0] 
```should work and why you can`t just extend from another interface or define these properties here like before

Reply:
i moved this repeating types to common interface `CourseSectionHandlers` and extend both components from it